packages/currencies-number/scripts/genCurrencies.mjs

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console, functional/no-expression-statement */
```

Comment:
This file is hacky, I made it a script rather than part of a package because it doesn't seem like it fits into the normal purpose of the library, its just a helper to reduce duplication of the currency code. I did check in the generated code, because I think it's important for it to be viewable to the end users.